CHANGELOG.md

```diff
@@ -48,6 +48,34 @@ release.
 - Clarify that `http/dup` has higher precedence than `http` in case both values are present
   in `OTEL_SEMCONV_STABILITY_OPT_IN`
   ([#249](https://github.com/open-telemetry/semantic-conventions/pull/249))
+- BREAKING: Generate System metrics semconv from YAML.
+  ([#89](https://github.com/open-telemetry/semantic-conventions/pull/89))
+  - Remove pluralization from process metrics `system.processes.*` to `system.process.*`
+  - Rename attributes for `system.cpu.*` metrics:
+    - `state` to `system.cpu.state`
+    - `cpu` to `system.cpu.logical`
+  - Rename attributes for `system.memory.*` metrics:
+    - `state` to `system.memory.state`
+  - Rename attributes for `system.paging.*` metrics:
+    - `state` to `system.paging.state`
+    - `type` to `system.paging.type`
+    - `direction` to `system.paging.direction`
+  - Rename attributes for `system.disk.*` metrics:
+    - `device` to `system.device`
+    - `direction` to `system.disk.direction`
+  - Rename attributes for `system.filesystem.*` metrics:
+    - `device` to `system.device`
+    - `state` to `system.filesystem.state`
+    - `type` to `system.filesystem.type`
+    - `mode` to `system.filesystem.mode`
+    - `mountpoint` to `system.filesystem.mountpoint`
+  - Rename attributes for `system.network.*` metrics:
+    - `device` to `system.device`
+    - `direction` to `system.network.direction`
+    - `protocol` to `network.protocol`
+    - `state` to `system.network.state`
+  - Rename attributes for `system.processes.*` metrics:
+    - `status` to `system.processes.status`
```

Comment:
Should be `system.process.status`, right?

Reply:
Yes, thanks for spotting it. It was also wrong in other places. Fixed it everywhere.